subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h

```diff
@@ -0,0 +1,133 @@
+ * Capture event timer on Radio end:
+ * wire the RADIO EVENTS_END event to the
+ * EVENT_TIMER TASKS_CAPTURE[<radio end timer>] task.
+ */
+#define HAL_RADIO_END_TIME_CAPTURE_PPI 11
+
+/*******************************************************************************
+ * Start event timer on RTC tick:
+ * wire the RTC0 EVENTS_COMPARE[2] event to EVENT_TIMER  TASKS_START task.
+ */
+#define HAL_EVENT_TIMER_START_PPI 7
+
+/*******************************************************************************
+ * Capture event timer on Radio ready:
+ * wire the RADIO EVENTS_READY event to the
+ * EVENT_TIMER TASKS_CAPTURE[<radio ready timer>] task.
+ */
+#define HAL_RADIO_READY_TIME_CAPTURE_PPI 8
+
+/*******************************************************************************
+ * Trigger encryption task upon address reception:
+ * wire the RADIO EVENTS_ADDRESS event to the CCM TASKS_CRYPT task.
+ *
+ * Note: we do not need an additional PPI, since we have already set up
+ * a PPI to publish RADIO ADDRESS event.
+ */
+#define HAL_TRIGGER_CRYPT_PPI HAL_RADIO_RECV_TIMEOUT_CANCEL_PPI
+
+/*******************************************************************************
+ * Trigger automatic address resolution on Bit counter match:
+ * wire the RADIO EVENTS_BCMATCH event to the AAR TASKS_START task.
+ */
+#define HAL_TRIGGER_AAR_PPI 12
+
+/*******************************************************************************
+ * Trigger Radio Rate override upon Rateboost event.
+ */
+#define HAL_TRIGGER_RATEOVERRIDE_PPI 13
+
+/******************************************************************************/
+#define HAL_ENABLE_PALNA_PPI 5
+
+#if defined(HAL_RADIO_FEM_IS_NRF21540)
+#define HAL_DISABLE_PALNA_PPI 4
+#else
+#define HAL_DISABLE_PALNA_PPI HAL_ENABLE_PALNA_PPI
+#endif
+
+#define HAL_ENABLE_FEM_PPI 3
+#define HAL_DISABLE_FEM_PPI HAL_DISABLE_PALNA_PPI
+
+/******************************************************************************/
+#if !defined(CONFIG_BT_CTLR_TIFS_HW)
+/* DPPI setup used for SW-based auto-switching during TIFS. */
+
+/* Clear SW-switch timer on packet end:
+ * wire the RADIO EVENTS_END event to SW_SWITCH_TIMER TASKS_CLEAR task.
+ *
+ * Note: we do not need an additional PPI, since we have already set up
+ * a PPI to publish RADIO END event.
+ */
+#define HAL_SW_SWITCH_TIMER_CLEAR_PPI HAL_RADIO_END_TIME_CAPTURE_PPI
+
+/* Wire a SW SWITCH TIMER EVENTS_COMPARE[<cc_offset>] event
+ * to a PPI GROUP TASK DISABLE task (PPI group with index <index>).
+ * 2 adjacent PPIs (8 & 9) and 2 adjacent PPI groups are used for this wiring;
+ * <index> must be 0 or 1. <offset> must be a valid TIMER CC register offset.
+ */
+#define HAL_SW_SWITCH_GROUP_TASK_DISABLE_PPI_BASE 14
+
+/* Enable the SW Switch PPI Group on RADIO END Event.
+ *
+ * Note: we do not need an additional PPI, since we have already set up
+ * a PPI to publish RADIO END event.
+ */
+#define HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI HAL_SW_SWITCH_TIMER_CLEAR_PPI
```

Comment:
Remove this from old code... seems to have been missed?

Reply:
Removed. Just learned that there is no warning when define is repeated with the same value.